File: app/src/main/java/com/desafioconcrete/jonasmferreira/appgitjavapop/domain/Owner.java
package com.desafioconcrete.jonasmferreira.appgitjavapop.domain;

import com.google.gson.annotations.SerializedName;

public class Owner {
    @SerializedName("id")
    long ID;

    String login;

    @SerializedName("avatar_url")
    String avatarUrl;
    String name;

    public long getID() {
        return ID;
    }

    public String getLogin() {
        return login;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public String getName() {
        return name;
    }
}

File: app/src/main/java/com/haldny/githubapp/domain/model/Owner.kt
package com.haldny.githubapp.domain.model

import com.google.gson.annotations.SerializedName

data class Owner(@SerializedName("id") val id: Int,
                 @SerializedName("login") val login: String,
                 @SerializedName("avatar_url") val avatarUrl: String)